packages/adblocker/src/filters/network.ts

```diff
@@ -1525,7 +1534,7 @@ export default class NetworkFilter implements IFilter {
 
     if (this.denyallow !== undefined) {
       if (this.denyallow.parts !== undefined) {
-        options.push(`denyallow=${this.denyallow.parts}`);
+        options.push(`denyallow=${this.denyallow.parts.replace(/,/g, '|')}`);
```

Comment:
Should be fixed in separate PR

Reply:
fixed in 86e5d84d14ab6b3bad55c158ae1f5df6ffbd1fb0